README.md

```diff
@@ -1,13 +1,184 @@
-# Kin core Stellar SDK for Android
+# Kin core SDK for Android
 Android library responsible for creating a new Stellar account and managing KIN balance and transactions.
 ![Kin Token](kin_android.png)
 
-## Disclaimer
+## Build
+
+* Add this to your module's `build.gradle` file.
+```gradle
+repositories {
+    ...
+    maven {
+        url 'https://jitpack.io'
+    }
+}
+...
+dependencies {
+    ...
+
+    compile "com.github.kinfoundation:kin-core-android:<latest version>"
+}
+```
+
+## Usage
+### Connecting to a service provider
+Create a new `KinClient` with two arguments: an android `Context` and a `ServiceProvider`. 
+
+A `ServiceProvider` provides details of how to access the Stellar horizon end point.
+The example below creates a `ServiceProvider` that will be used to connect to the main (production) Stellar 
+network
+```java
+ServiceProvider infuraProvider =  
+    new ServiceProvider("https://horizon.stellar.org", ServiceProvider.NETWORK_ID_MAIN));
+KinClient kinClient = new KinClient(context, infuraProvider);
+```
+
+To connect to a test Stellar network use the following ServiceProvider:
+```java
+new ServiceProvider("https://horizon-testnet.stellar.org", ServiceProvider.NETWORK_ID_TEST)
+``` 
+
+### Creating and retrieving a KIN account
+The first time you use `KinClient` you need to create a new account, using a passphrase. 
+The details of the account created will be securely stored on the device.
+```java
+KinAccount account;
+try {
+    if (!kinClient.hasAccount()) {
+        account = kinClient.addAccount("yourPassphrase");
+    }
+} catch (CreateAccountException e) {
+    e.printStackTrace();
+}
+```
+
+Once an account has been created there is no need to call `createAccount` again on the same device. 
+From then on calling `getAccount` will retrieve the account stored on the device.
+```java
+if (kinClient.hasAccount()) {
+    account = kinClient.getAccount(0);
```

Comment:
explain you need a 0 parameter

Reply:
yeah, I revised a bit the documentation, to reflect we have multiple accounts and index should be provided.